examples/chat-example/src/main.rs

```diff
@@ -149,6 +163,9 @@ async fn main() -> Result<(), Box<dyn Error>> {
                         "Got message: '{}' with id: {id} from peer: {peer_id}",
                         String::from_utf8_lossy(&message.data),
                     ),
+                SwarmEvent::NewListenAddr { address, .. } => {
+                    println!("Local node is listening on {:?}", address);
```

Comment:
```suggestion
                    println!("Local node is listening on {address}");
```

This will be nicer to look at.